MinimedKitUI/MinimedPumpSettingsViewController.swift

```diff
@@ -187,6 +190,13 @@ class MinimedPumpSettingsViewController: RileyLinkSettingsViewController {
             case .preferredInsulinDataSource:
                 cell.textLabel?.text = LocalizedString("Preferred Data Source", comment: "The title text for the preferred insulin data source config")
                 cell.detailTextLabel?.text = String(describing: pumpManager.preferredInsulinDataSource)
+            case .useMySentry:
+                cell.textLabel?.text = LocalizedString("Use MySentry", comment: "The title text for the preferred MySentry setting config")
+                if pumpManager.state.pumpModel.hasMySentry {
```

Comment:
This line will never evaluate to false, right? Pumps that do not have mySentry will not have this row. This conditional can be removed, I think.

Reply:
Done, thanks.